### PR TITLE
(IRO-2519) Your Node review fixes.

### PR DESCRIPTION
**Pull request by azhurauski-sc**

### CODE REVIEW

**Review by skylarrichard123 — CHANGES_REQUESTED** [review]
Just noticed one more thing:
https://linear.app/ironfish/issue/IRO-2529/remove-padding

Everything else looks great

**Review by skylarrichard123 — APPROVED** [review]

File: src/hooks/node/useNodeStatus.ts [hunk 1]
```diff
@@ -1,6 +1,15 @@
...
 
+interface NodeStatusResponse extends GetStatusResponse {
+  node: {
+    status: 'started' | 'stopped' | 'error'
+    version: string
+    git: string
+    nodeName: 'My Node Name Example'
+  }
+}
+
```
Comment (ndemidovich-sc) [inline src/hooks/node/useNodeStatus.ts:12]: Do we need this in feature?

**Review by ndemidovich-sc — APPROVED** [review]

File: src/hooks/node/useNodeStatus.ts (hunk 1 above)
Comment (azhurauski-sc) [reply src/hooks/node/useNodeStatus.ts:12]: Not sure but there is no nodeName in GetStatusResponse, will add a comment

Comment (ndemidovich-sc) [reply src/hooks/node/useNodeStatus.ts:13]: https://github.com/iron-fish/ironfish/blob/f1dcebb99484d5a1dce940eedf0f32918699edd2/ironfish/src/rpc/routes/node/getStatus.ts#L20
Possibly you are not update dependencies

**Pull request merged**